### DISCUSSION TITLE
Projects Beta: No way to restore project/board task/issues

### DISCUSSION THREAD

**Discussion by jontingvold** [community/community#12807] [Projects and Issues]
On the old projects boards there was a way to restore "archived" tasks/issues by clicking on "Menu" → "..." → "View archive".

In the new projects (beta) there is a button to "archive" tasks/issues but no way to restore them. At least I can't find any menu with "View archive" or similar anywhere. In fact the "archive" action, not practically deletes the task/issue and should in fact be renamed "Delete".

**Comment by serdarkalayci** [comment]
If you have a related issue, you can see related card as archived there, and you can restore it. 
![image](https://user-images.githubusercontent.com/14258482/158175667-88f74023-3f50-4982-8fbb-9a0e7fab48cf.png)
But I also could not find a way to list/view archived card from the project view.

**Comment by cognifloyd** [comment]
I found this in the docs:
> To restore an archived item, navigate to the issue or pull request. In the project side bar on the issue or pull request, click Restore for the project that you want to restore the item to. Draft issues cannot be restored.

https://docs.github.com/en/issues/trying-out-the-new-projects-experience/creating-a-project#restoring-archived-items

Sadly, that does not help me, because I have a personal project that I'm tracking issues on many different repos. And I'm only a contributor to those repos, not a maintainer, so I can't manage the project from that issue.
So, once I archive an issue, I cannot un-archive it, and I cannot re-add it (because "Content already exists in this project" according to error message).
